addons/test_mail/tests/test_mail_message.py

```diff
@@ -23,6 +24,41 @@ def setUpClass(cls):
         })
         cls.Message = cls.env['mail.message'].with_user(cls.user_employee)
 
+    @users('employee')
+    def test_message_update_content_bus(self):
```

Comment:
Tests about channels should not be in `test_mail` but in `mail` (`discuss` subfolder)

Reply:
Anyway it is redundant with existing tests, don't see what it tests.

Reply:
Oh it's the message update... ok then ^^